sdk/communication/communication-call-automation/recordings/CallConnection_Live_Tests_Add_a_participant_and_get_call_properties.json

```diff
@@ -1,4 +1,28 @@
 [
+  [
```

Comment:
We should migrate recordings out of this repo. Please refer to this guide: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/recorder/ASSET_SYNC_MIGRATION.md#migration